contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -49,32 +48,27 @@ def main(argv):
     test_cases = [
         LoadCapture(capture_file_path="testdata\\OrbitTest_1-64.orbit", expect_fail=True),
         LoadCapture(capture_file_path="testdata\\OrbitTest_1-72.orbit"),
-
         FilterTracks(filter_string="Scheduler", expected_track_count=1),
         CheckTimers(track_name_filter='Scheduler*'),
-
         FilterTracks(filter_string="Frame", expected_track_count=1),
         CheckTimers(track_name_filter='Frame track*'),  # Verify the frame track has timers
-
         FilterTracks(filter_string="DynamicName_", expected_track_count=5),
-
         FilterTracks(filter_string="_var", expected_track_count=6),
-
         FilterTracks(filter_string="OrbitThread_", expected_track_count=1),
         ToggleCollapsedStateOfAllTracks(),
         CheckTimers(track_name_filter="OrbitThread_*"),
         CheckThreadStates(track_name_filter='OrbitThread_*'),
-
         FilterTracks(filter_string="ORBIT_ASYNC_TASKS", expected_track_count=1),
         CheckTimers(track_name_filter="ORBIT_ASYNC_TASKS"),
         FilterTracks(filter_string="ORBIT_START_ASYNC_TEST", expected_track_count=1),
         CheckTimers(track_name_filter="ORBIT_START_ASYNC_TEST"),
-
         FilterTracks(filter_string=""),
         VerifyTracksExist(track_names=["Page*", "*System*", "*CGroup*"], allow_duplicates=True),
-
         AddIterator(function_name="TestFunc2"),
-        VerifyFunctionCallCount(function_name="TestFunc2", min_calls=1257, max_calls=1257),
+        VerifyScopeTypeAndHitCount(scope_name="TestFunc2",
+                                   scope_type="D",
+                                   min_calls=1257,
+                                   max_calls=1257),
```

Comment:
Soo, same problem as before, you renamed these parameters. Same for other files.

Reply:
Yeah, I've noticed this.  This time it makes the code fail.  I guess, I shouldn't expect F2 to rename it all.

Reply:
Depends on the IDE I guess. `PyCharm` worked generally fine for me on this.